site/src/modules/tasks/TaskDeleteDialog/TaskDeleteDialog.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { MockTasks } from "testHelpers/entities";
+import { withGlobalSnackbar } from "testHelpers/storybook";
+import type { Meta, StoryObj } from "@storybook/react-vite";
+import { API } from "api/api";
+import { expect, spyOn, userEvent, waitFor, within } from "storybook/test";
+import { TaskDeleteDialog } from "./TaskDeleteDialog";
+
+const meta: Meta<typeof TaskDeleteDialog> = {
+	title: "modules/tasks/TaskDeleteDialog",
+	component: TaskDeleteDialog,
+	decorators: [withGlobalSnackbar],
+};
+
+export default meta;
+type Story = StoryObj<typeof TaskDeleteDialog>;
+
+export const DeleteTaskSuccess: Story = {
+	decorators: [withGlobalSnackbar],
+	args: {
+		open: true,
```

Comment:
Do we also need to pass in `task` and `onClose` since those are non-optional args?

Reply:
Ah yeah I noticed there is no actual enforcement of the types in Storybook tests, but really seems like there should be.  :thinking:  Seems weird to discard type safety for tests.